src/pages/gd/Claim/index.tsx

```diff
@@ -1,52 +1,25 @@
 import React, { memo } from 'react'
 import { t } from '@lingui/macro'
 import { useLingui } from '@lingui/react'
-// TODO: check when good-design will be merged
 // import { ClaimButton } from '@gooddollar/good-design'
-import { StyleSheet } from "react-native";
-import { useTheme } from "styled-components";
-import { Text, View } from "native-base";
+import { NativeBaseProvider } from 'native-base'
+import { ClaimButton, theme } from '@gooddollar/good-design'
 
 const Claim = () => {
-	const {i18n} = useLingui()
-	const theme = useTheme()
-	const styles = makeStyles(theme)
-
-	return (
-		<View style={styles.layout}>
-			<Text color={theme.color.text4}>
-				{i18n._(t`Claim`)}
-			</Text>
-			<View style={styles.content}>
-				{/*<ClaimButton />*/}
-			</View>
-		</View>
-	)
+    const { i18n } = useLingui()
+    return (
+        <NativeBaseProvider theme={theme}>
```

Comment:
@alt-a1r any styles should be defined either in the theme, or specifically styled on the component returned